.gitignore

```diff
@@ -2,8 +2,38 @@ packwiz-installer-bootstrap.jar
 packwiz-installer.jar
 completion.sh
 reload.bat
-# Exclude exported CurseForge zip files
-*.zip
+config/sodium*
+config/iris*
+.fabric
+backups
+coremods
+crash-reports
+ESM
+fancymenu*
+!fancymenustuff
+item_renderer
+local
+logs
+saves
+screenshots
+ModTranslations
+observable_profiles
+rei_exports
+shaderpacks
+shaderpacks/*
+texturepacks
+hs_err_pid*
+o_prof
+imgui.log
+ears_debug.log
+hotbar.nbt
+rhino.local.properties
+servers.dat*
+usercache.json
+journeymap
+!journeymap/defaultconfig.json
+mods
```

Comment:
Why add the mods directory? That will prevent anyone else in the future making changes to files inside of there. That's not what we want. You should just not add the jars to the git commits.